### DISCUSSION TITLE
increase rpiCameraBitrate seems do not take effect

### DISCUSSION THREAD

**Discussion by marcob79** [bluenviron/mediamtx#1748] [Questions]
Hi all, I was trying to increase rpiCameraBitrate from 1000000 to 15000000 but the stream bitrate seems always around 1Mbps.
I have a picamera v3 in fullHD (1920x1080) @ 30fps.

anyone succeeded in increasing desired bitrate of picamera?

**Comment by aler9** [comment]
Hello, the `rpiCameraBitrate` parameter is passed directly to the hardware encoder, there isn't any bitrate-related code into the server. From my experience, this parameter acts as upper limit: the encoder scales down the video quality in order to stay below it.
If the bitrate stays the same after increasing `rpiCameraBitrate`, it means that the quality is already the maximum available from the specific sensor.

[a comment by an automated account is not shown]
